test/README.md

```
@@ -35,6 +35,13 @@ So for instance to run in discovery mode the command line could look like this:
 KUBECONFIG="/home/user/.kube/config" PTP_TEST_MODE=Discovery make functests
 ```
 
+To exercise BC reverse-sync `os-clock-sync-state` FREERUN (serial outage recovery test) with events:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)